run/mainnet/docker/.env

```
@@ -0,0 +1 @@
+NETWORK="mainnet"
```

```suggestion
NETWORK="mainnet"

```

(newline at end of file)